openmrs-19/src/main/java/org/motechproject/openmrs19/service/impl/OpenMRSPatientServiceImpl.java

```diff
@@ -93,20 +94,44 @@ public OpenMRSPatient getPatientByUuid(String uuid) {
 
         String motechId = null;
         OpenMRSFacility facility = null;
-        Identifier identifier = patient.getIdentifierByIdentifierType(motechIdentifierUuid);
+        Identifier motechIdentifier = patient.getIdentifierByIdentifierType(motechIdentifierUuid);
 
-        if (identifier == null) {
+        if (motechIdentifier == null) {
             LOGGER.warn("No MoTeCH Id found on Patient with id: " + patient.getUuid());
         } else {
 
-            if (identifier.getLocation() != null) {
-                facility = facilityAdapter.getFacilityByUuid(identifier.getLocation().getUuid());
+            if (motechIdentifier.getLocation() != null) {
+                facility = facilityAdapter.getFacilityByUuid(motechIdentifier.getLocation().getUuid());
             }
 
-            motechId = identifier.getIdentifier();
+            motechId = motechIdentifier.getIdentifier();
         }
 
-        return ConverterUtils.toOpenMRSPatient(patient, facility, motechId);
+        List<Identifier> supportedIdentifierList = new ArrayList<>();
```

Comment:
Can the process for discovering supported identifiers be moved to separate, private method?